src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -949,14 +949,15 @@ server.3=zoo3:2888:3888</programlisting>
           </varlistentry>
 
           <varlistentry>
-            <term>ttlNodesEnabled</term>
+            <term>zookeeper.extendedTypesEnabled</term>
 
             <listitem>
-              <para>(No Java system property)</para>
+                <para>(Java system property only: <emphasis
```

Comment:
This is a bit confusing given QuorumPeerConfig line 326

                System.setProperty("zookeeper." + key, value);

See the doc for forceSync for an example of a better way to set this up. (forceSync works this same way)